package.json

```diff
@@ -2,20 +2,24 @@
   "scripts": {
     "build": "node build.cjs",
     "fmt": "prettier --write .",
-    "test": "node ./dist/tests.js"
-  },
-  "devDependencies": {
-    "esbuild": "^0.8.31",
-    "prettier": "^2.2.1"
+    "test": "node ./dist/tests.js",
+    "go": "yarn build && ts-node ./src/tests.ts"
   },
   "dependencies": {
     "@verto/flex": "^0.1.12",
-    "arlocal": "^1.1.16",
     "arweave": "^1.11.4",
     "crocks": "^0.12.4",
     "replace-in-file": "^6.3.5",
     "smartweave": "^0.4.49",
-    "warp-contracts": "^1.1.14"
+    "warp-contracts": "1.2.0-rc.2"
```

Comment:
this is basically the same as the rc.1 - with the web bundlers - related fixed (more details here: https://github.com/warp-contracts/bundlers)